action.yml

```diff
@@ -7,6 +7,8 @@ inputs:
     default: 'false'
   node-version:
     description: 'Version Spec of the version to use.  Examples: 12.x, 10.15.1, >=10.15.0'
+  node-arch:
+    description: 'Target architecture for Node to use. Examples: x86, x64. Will use system architecture by default.'
```

Comment:
Please rename the `node-arch` input to the `architecture` in order to be consistent with the [actions/setup-python](https://github.com/actions/setup-python/blob/main/action.yml#L9) action

Reply:
Fixed in https://github.com/actions/setup-node/pull/194/commits/7d4b86e7a4f1bb493dd4314fad249fb481194f6b